# ID que define de qual robô é a comando
uint32 id

# Velocidade da roda esquerda
float32 left_vel

# Velocidade da roda direita
float32 right_vel
